extensions/amp-browsi/0.1/test/validator-amp-browsi.html

```diff
@@ -0,0 +1,59 @@
+<!doctype html>
```

Comment:
Please add the license with 2019 as the date to this file.

Also add a test description of what this file is testing. For an example see this [test file](https://github.com/ampproject/amphtml/blob/master/extensions/amp-inputmask/0.1/test/validator-amp-inputmask.html).

Reply:
@honeybadgerdontcare can you help me understand why 'gulp validator' is always failing?
when local, I get these error lines:
```
google.protobuf.descriptor not found. Try "pip install protobuf" or follow the install instructions at https://github.com/ampproject/amphtml/blob/master/validator/README.md#installation
[11:23:58] The following tasks did not complete: validator
[11:23:58] Did you forget to signal async completion?
```